webapp/channels/src/components/autocomplete_selector.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('components/widgets/settings/AutocompleteSelector', () => {
             />,
         );
 
-        wrapper.instance().onBlur();
+        (wrapper.instance() as AutocompleteSelector).onBlur();
```

Comment:
You should be able to pass the type directly to the shallow, and avoid these castings:

```
const wrapper = shallow<AutocompleteSelector>(
```

Reply:
Thank you, I missed this.